### ISSUE TITLE
Error running Jenny: Could not load type 'Jenny.CodeGenFile'

### ISSUE THREAD

**Issue by maximiliancsuk** [sschmid/Entitas-Standalone#6]
Hi,

I'm trying to get this sample project to run, but get errors from a freshly cloned repository.

First, when starting the Jenny Server:
```
dotnet run -c Release --project jenny/MyProject.Jenny.csproj
    × Could not resolve: C:\Users\maxim\Projects\games\Entitas-Standalone\jenny\bin\Release\net6.0\MyProject.Jenny.exe
Listening on port 3333
```

Not sure about the `Could not resolve` line, but the server seems to start at least.

Then, when trying to generate the code:

```
dotnet jenny/bin/Release/net6.0/MyProject.Jenny.dll client gen
    × Could not resolve: C:\Users\maxim\Projects\games\Entitas-Standalone\jenny\bin\Release\net6.0\MyProject.Jenny.exe
Could not load type 'Jenny.CodeGenFile' from assembly 'Jenny, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.Could not load type 'Jenny.CodeGenFile' from assembly 'Jenny, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
```

Nothing gets generated.

Any tips?

Not sure if this is related to #5 , I tried a few things with the attached .nupkg file, but no success.

**Comment by sschmid** [comment]
@maximiliancsuk 
I just fixed and closed #5 

Could you update and try again?

**Comment by maximiliancsuk** [comment]
Updated and tried again: works perfect now, thank you!
